src/ABSTRACT.md

```diff
@@ -38,7 +38,7 @@ The optimised Plonkish constraint system specification will detail:
 - Support for rotation constraints, which leverage the relative positions of assigned matrix elements to reduce the size of the matrix and the number of equality constraints.
 - How to compile to the optimised constraint system.
 
-Accompanying the specification there will be an open-source reference implementation written in Rust, and corresponding test vectors.  We also intend to include an executable specification of the Plonkish relation, to facilitate formally verifying that the optimisations included in the optimised constraint system efficiently preserve the completeness, knowledge soundness, and zero-knowledge properties of proof systems applied to the unoptimised constraint system.
+Accompanying the specification there will be an open-source reference implementation written in Rust, and corresponding test vectors.  We also intend to include an executable specification of the Plonkish relation in Lean 4, to facilitate formally verifying that the optimisations included in the optimised constraint system efficiently preserve the completeness, knowledge soundness, and zero-knowledge properties of proof systems applied to the unoptimised constraint system.
```

Comment:
It might not be the case that the Lean 4 impl is the executable specification (depends what we mean by "executable" I guess), but that we have an impl in `plonkish-relation` that we can prove equivalent to the Lean 4 spec.